### PR TITLE
Document doc-validator

### PR DESCRIPTION
**Pull request by jdbaldry**
- Fix quotation of shell arguments
- Document `doc-validator` Make target
- Remove `doc-validator/%` target

### CODE REVIEW

**Review by knylander-grafana — APPROVED** [review]

**Review by josmperez — APPROVED** [review]
I just gave it a quick look over and it looks good. I wonder though if 'Errata' is descriptive enough as a page title. Is this topic intended to consolidate all errata for all doc tools and workflows into one page?

File: docs/sources/writing-guide/tooling-and-workflows/validate-technical-documentation/errata.md [hunk 1]
```diff
@@ -0,0 +1,306 @@
+---
+title: Errata
```
Comment (jdbaldry) [inline docs/sources/writing-guide/tooling-and-workflows/validate-technical-documentation/errata.md:2]:
```suggestion
title: Errata for doc-validator
```

File: docs/sources/writing-guide/tooling-and-workflows/validate-technical-documentation/errata.md [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+description: Error codes and descriptions for doc-validator errata.
+---
+
+# Errata
```
Comment (jdbaldry) [inline docs/sources/writing-guide/tooling-and-workflows/validate-technical-documentation/errata.md:6]: Errata for doc-validator

**Review by jdbaldry — COMMENTED** [review]
Add more descriptive title.

File: docs/sources/writing-guide/tooling-and-workflows/validate-technical-documentation/errata.md [hunk 3]
```diff
@@ -0,0 +1,306 @@
+---
+title: Errata for doc-validator
```
Comment (jdbaldry) [inline docs/sources/writing-guide/tooling-and-workflows/validate-technical-documentation/errata.md:2]:
```suggestion
title: Errata for doc-validator
menuTitle: Errata
```

**Comment by jdbaldry** [comment]
> I just gave it a quick look over and it looks good. I wonder though if 'Errata' is descriptive enough as a page title. Is this topic intended to consolidate all errata for all doc tools and workflows into one page?

Good point, I've add some more context to the heading and the title for that page to make it clear that it is only `doc-validator` errata.

**Pull request merged**